catroid/src/main/java/org/catrobat/catroid/soundrecorder/SoundRecorder.java

```diff
@@ -38,12 +38,16 @@
 
 	private String path;
 
+	public SoundRecorder() {
+		this("/dev/null");
+	}
+
 	public SoundRecorder(String path) {
 		this.recorder = new MediaRecorder();
 		this.path = path;
 	}
 
-	public void start() throws IOException, RuntimeException {
+	public void start() throws IOException, IllegalStateException {
```

Comment:
This method, is a bit odd, why bother to catch an exception if it's thrown in the same catch clause? 
Please explain your change here :)